libr/include/r_core.h

```diff
@@ -369,7 +381,7 @@ struct r_core_t {
 	bool marks_init;
 	ut64 marks[UT8_MAX + 1];
 	RThreadChannel *chan; // query
-	RThreadLock *lock;
+	RThreadLock *lock;	
```

Comment:
trailing tabs?

Reply:
ill add a linter for this but pls merge this fix
```suggestion
	RThreadLock *lock;
```